### PR TITLE
Remove read delay

### PR DESCRIPTION
**Pull request by sebastianlavender**
- removed read time delay
- refactored OffsetFetcher to GlobalLatestOffsetCache
- updated tests to reflect changes

### CODE REVIEW

File: pipe-storage-postgresql/src/main/java/com/tesco/aqueduct/pipe/storage/PostgresqlStorage.java [hunk 1]
```diff
@@ -150,7 +150,7 @@ private MessageResults readMessages(
         connection.setAutoCommit(false);
         setWorkMem(connection);
 
-        final long globalLatestOffset = offsetFetcher.getGlobalLatestOffset(connection);
+        final long globalLatestOffset = globalLatestOffsetCache.getGlobalLatestOffset(connection);
```
Comment (ashish-sharma09) [inline pipe-storage-postgresql/src/main/java/com/tesco/aqueduct/pipe/storage/PostgresqlStorage.java:153]: Feels like `globalLatestOffsetCache.get(connection)` is probably good enough

**Review by ashish-sharma09 — APPROVED** [review]
LGTM

File: pipe-storage-postgresql/src/main/java/com/tesco/aqueduct/pipe/storage/PostgresqlStorage.java (hunk 1 above)
Comment (sebastianlavender) [reply pipe-storage-postgresql/src/main/java/com/tesco/aqueduct/pipe/storage/PostgresqlStorage.java:153]: Yeah I think you're right - I was debating that the class name and method seemed too long

**Comment by LorenzoCianciaruso** [comment]
read-delay-seconds needs to be removed from the application.yamls too

**Review by LorenzoCianciaruso — APPROVED** [review]

**Pull request merged**